training/MNISTFashionMicroservice/src/server/training.py

```diff
@@ -2,20 +2,211 @@
 import tempfile
 import shutil
 import requests
+import sys
+import logging
+import json
 
 from src.server.dependency import ModelData
 import tensorflow as tf
 
+class StreamToLogger(object):
+    """
+    Fake file-like stream object that redirects writes to a logger instance.
+    Source: https://stackoverflow.com/a/39215961
+    """
+    def __init__(self, logger, level):
+       self.logger = logger
+       self.level = level
+       self.linebuf = ''
+
+    def write(self, buf):
+       for line in buf.rstrip().splitlines():
+          self.logger.log(self.level, line.rstrip())
+
+    def flush(self):
+        pass
+
+# def train_model(training_id, model_data: ModelData):
```

Comment:
feel free to remove commented code

Reply:
Ok, thanks! I will remove the commented code out and make another pull request. 

For the example/test case, I updated the Postman collection URL in the README to a new one which has an example in "Create Training Request" API.

Reply:
I think you can just update this pull request and I should be able to see the changes

Reply:
okay! got it.

Reply:
I updated the README file and removed the commented code out. Thanks